docs/lib/govuk_tech_docs/open_api/templates/api_reference_full.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= Rouge::Themes::Github.render(scope: '.highlight') %>
 </style>
 
-<h1 id="<%= info.title.parameterize %>">Endpoints <%= document.info.version %></h1>
+<h1 id="<%= info.title.parameterize %>"><%= document.info.title %></h1>
```

Comment:
```suggestion
<%= tag.h1(document.info.title, id: info.title.parameterize) %>
```

Reply:
i'll work this into the next PR

Reply:
i just tried this and don't have access to `tag`. I'm guessing this is a rails view helper? If so unfortunately we're not in a rails app not sure where to grab this helper from?